themes/hugo_theme_robust/layouts/index.html

```diff
@@ -6,7 +6,22 @@
   <div class="row">
     <div class="col-md-9">
 
-      <h1 class="p-page-title">R Training</h1>
+      <h1 class="p-page-title">R Training Curriculum</h1>
+      
+      <p>
+        This is a collection of the USGS R training curriculum, and materials that correspond to specific workshops. Currently, there is only one course - <i>Introduction to R</i>. The curriculum used throughout this workshop is available here. Navigate using the side menu or the "Next/Prev" buttons on each lesson. 
```

Comment:
@ldecicco-USGS can you review the text I put here. Once this is merged, I will reassign #48 to @mwernimont for some styling